Veeresh/HTML5/Assignment1.html

```diff
@@ -0,0 +1,153 @@
+<html>
+<head>
+    <title>HTML5 Assignment 1</title>
+    <style>
+        body {
+            background-color: hsl(154, 100%, 35%);
+        }
+
+        #form {
+            background-color: white;
+            max-width: 400px;
+            display: block;
+            margin: auto;
+            padding-top: 25px;
+        }
+
+        #label_name,
+        #label_email,
+        #label_age,
+        #label_describe,
+        #label_radio,
+        #label_checkbox,
+        #label_comments_suggestion {
+            padding-left: 20px;
+            font-size: 16px;
+        }
+
+        #checkbox,
+        #radiobox {
+            font-size: 16px;
+            margin-left: 8px 0px;
+            margin-top: 8px;
+            padding: 0px 3px;
+        }
+
+        input[type=checkbox],
+        input[type=radio] {
+            margin-left: 20px;
+            margin-top: 8px;
+        }
+
+        #name,
+        #email,
+        #age {
+            width: 350px;
+            height: 20px;
+            display: inline-block;
+            border: 1px solid rgb(71, 70, 70);
+            border-radius: 4px;
+            margin-left: 20px;
+            margin-bottom: 8px;
+            padding-bottom: 15px;
+            padding-top: 20px;
+            margin-top: 8px;
+            padding-left: 15px;
+
+        }
+
+        #describe {
+            padding: 7px 125px;
+            margin-left: 20px;
+            border: 1px solid rgb(71, 70, 70);
+            border-radius: 4px;
+            margin-top: 8px;
+        }
+
+        h3 {
+            text-align: center;
+        }
+
+        #submit {
+            margin-bottom: 20px;
+            border: 1px solid;
+            border-radius: 4px;
+            width: 350px;
+            height: 25px;
+            margin-left: 20px;
+            text-align: center;
+            font-size: 16px;
+            cursor: pointer;
+            background-color: hsl(154, 100%, 35%);
+
+        }
+
+        #comments_suggestion {
+            margin-left: 20px;
+            margin-top: 8px;
+        }
+    </style>
+</head>
+
+<body>
+    <h3>Anguler Form</h3>
+    <div id="form">
+        <form action="">
+            <label for="name" id="label_name"> Name </label><br>
+            <input type="text" name="name" id="name" placeholder="Enter Your Name" autocomplete="off"><br><br>
+            <label for="email" id="label_email"> Email </label><br>
+            <input type="email" name="name" id="email" placeholder="Enter Your Email" autocomplete="off"><br><br>
```

Comment:
change as inp_email. Similarly change for others. Try to standardize the naming convention.